charts/geoipupdates/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: MaxMind GeoIP database updater
 name: geoipupdates
-version: 1.0.0
+version: 1.0.1
```

```suggestion
version: 2.0.0
```

It is a breaking change....